src/services/location-service.ts

```diff
@@ -0,0 +1,45 @@
+import { distanceBetweenTwoPoints } from '@/services/util/DistanceCalculator';
+
+const SKOOKUM_COORDINATES = {
+  latitude: 39.758117, // default to skookum coordinates
+  longitude: -104.989888, // default to skookum coordinates
+};
+
+export class LocationService {
+
+  /**
+   * Request permission to get the user's location while the
+   * browser page is active.
+   */
+  public requestLocation(): Promise<any> {
```

Comment:
This guy will need to be beefed up depending on the device and current browser permissions.